integrationtest/mapper/pojo/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/mapping/definition/DocumentIdBaseIT.java

```diff
@@ -10,7 +10,6 @@
 
```

Comment:
I think we should change `@MarkerMappingBuilderReference`  too. To keep this API consistent with the others. I'll do in a different commit, so that we could easly remove it in case.